guides/source/debugging_rails_applications.md

```diff
@@ -461,11 +490,11 @@ the `thread` command (or the abbreviated `th`). This command has a handful of
 options:
 
 * `thread`: shows the current thread.
-* `thread list`: is used to list all threads and their statuses. The plus +
-character and the number indicates the current thread of execution.
-* `thread stop _n_`: stop thread _n_.
-* `thread resume _n_`: resumes thread _n_.
-* `thread switch _n_`: switches the current thread context to _n_.
+* `thread list`: is used to list all threads and their statuses. The current
+thread is marked with a plus (+) sign.
+* `thread stop n`: stops thread _n_.
+* `thread resume n`: resumes thread _n_.
+* `thread switch n`: switches the current thread context to _n_.
```

Comment:
> Yes, but being consistent it more important. So we will change accordingly.

That's why I wrapped two places to 80 chars, because this whole section of the guide (except for those 2 places) is wrapped to 80 chars.

> Looks like there has been confusion in how it was used previously http://take.ms/ZKf6v . The first _ is not useful when wrapped in quote, everything outside quotes serves its purpose.
@deivid-rodriguez lets change to use how you have used here, wherever its wrapped in quotes(`), unless the command itself has [n]. Everything outside remains the same.

Those I think those are exactly the changes I'm including in this PR...